TombEngine/Renderer/Renderer11.cpp

```diff
@@ -495,9 +495,9 @@ namespace TEN::Renderer
 	{
 		D3D11_RECT rects;
 		rects.left = s.left;
-		rects.top = m_screenHeight - s.top;
```

Comment:
Were scissor tests successful after that change? I remember doing the same "fix" you did there, but it badly messed tests and made them work even more incorrectly.

Reply:
From test it seems fine. Now rooms collector works in clip space, so -1 is bottom of screen that's why we need to do this operation. I will polish this a bit before the release.